aya/src/generated/mod.rs

```diff
@@ -3,6 +3,8 @@
 mod btf_internal_bindings;
 #[cfg(target_arch = "aarch64")]
 mod linux_bindings_aarch64;
+#[cfg(target_arch = "arm")]
```

Comment:
shouldn't this be armv7?

Reply:
nope, for rust target_arm==armv7 https://doc.rust-lang.org/reference/conditional-compilation.html#target_arch